services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
Using mutable default arguments like `list` can lead to unexpected behavior. Consider using `None` as the default and initializing the list inside the function.